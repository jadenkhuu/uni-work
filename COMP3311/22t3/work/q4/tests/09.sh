psql racing -c "select * from q4('ss') order by horse;"
