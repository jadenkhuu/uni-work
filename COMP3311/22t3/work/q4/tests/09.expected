        horse        | average 
---------------------+---------
 Alanis Mare-issette |   54666
 Burress             |   50312
 David Hasselhoofs   |  129470
 Firdaussi           |   34736
 Isinglass           |    9368
 Passing Wind        |   13866
 Riding Miss Daisy   |   27058
 Russel              |    1411
(8 rows)
